brigade.js

```diff
@@ -134,20 +133,50 @@ function publish(e, p) {
   return goPublish;
 }
 
+// These represent the standard checks that run for PRs (see runSuite below)
+// 
+// They are encapsulated in an object such that individual checks may be run
+// using data supplied by a corresponding GitHub webhook, say, on a
+// check_run:rerequested event (see runCheck below)
+checks = {
+  "verify": { runFunc: verify, description: "Verify" },
+  "build": { runFunc: build, description: "Build" },
+  "crossplatformbuild": { runFunc: xbuild, description: "Cross-Platform Build" },
+  "test": { runFunc: test, description: "Test" },
+  "integrationtest": { runFunc: testIntegration, description: "Integration Test" }
+};
+
+// runCheck can be invoked to (re-)run an individual GitHub Check Run, as opposed to a full suite
+function runCheck(e, p) {
+  payload = JSON.parse(e.payload);
+
+  name = payload.body.check_run.name;
+  check = checks[name];
+
+  if (typeof check !== 'undefined') {
+    checkRun(e, p, check.runFunc, check.description)
+      .catch(e => {console.error(e.toString())});
+  } else {
+    err = new Error(`No check found with name: ${name}`);
+    // TODO: remove this console.error statement once Brigade logs err.message when thrown
```

Comment:
TODO's are like hydra, for everyone one you slay, another shall take its place. 😂 

jk, though, this is great 👍